packages/contentstack-config/src/commands/config/get/rate-limit.ts

```diff
@@ -0,0 +1,52 @@
+import { cliux, configHandler, isAuthenticated } from '@contentstack/cli-utilities';
+import { Command } from '@contentstack/cli-command';
+import { handleErrorMsg } from '../../../utils/interactive';
+import { RateLimitConfig } from '../../../interfaces';
+
+export default class RateLimitGetCommand extends Command {
+
+  static examples = ['$ csdx config:get:rate-limit'];
+
+  async run() {
+    try {
+      if (!isAuthenticated()) {
+        const err = { errorMessage: 'You are not logged in. Please login with command $ csdx auth:login' };
+        handleErrorMsg(err);
+      }
```

Comment:
@harshithad0703 , authentication isn't required for this command.

Reply:
removed authentication here